styleguide/_themes/global/scss/_modules.scss

```diff
@@ -1,28 +1,32 @@
 .subnav{
   background-color:$blue-gray;
   border-radius:0px;
-  button{
-    margin: 10px 0px;
+  .subnav-inside{
+    width: 100%;
+  }
+  button, .button{
```

Comment:
Every subnav has a CTA and I remember seeing a conversation about supporting both anchor tags and button tags last week so I was following that decision.